insta/src/components/Login/Logout.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const LogOut = props => {
```

Comment:
You could have just made this a button in the other file. But this works to. lol.